src/script/target/webpack/overrideConfiguration.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+
+module.exports = state => Object.assign({}, state.raw, {
+  entry: {
+    bundle: path.resolve(__dirname, '..', '..', '..', '..', 'dist', 'gui.build.js'),
```

Comment:
I will think about it.

In one hand I like your idea.
In other hand, I like the fact that each js file has its own concern and I don't like that the first function has to much concerns (set all path, trigger sub-functions, etc).